test/src/kbasesearchengine/test/events/storage/MongoDBStatusEventStorageTest.java

```diff
@@ -127,6 +127,7 @@ public void storeAndGetNoTypeEmptyCodesAllFields() throws Exception {
                 .withNullableAccessGroupID(6)
```

Comment:
Copying comment from previous review:
Shouldn't these tests check that false and null are stored correctly?

Reply:
Added tests.